### PR TITLE
Deprectae OC.Util.formatDate and OC.Util.relativeModifiedDate

### PR DESCRIPTION
**Pull request by rullzer**
Signed-off-by: Roeland Jago Douma <roeland@famdouma.nl>

### CODE REVIEW

**Comment by ChristophWurst** [comment]
> [ChristophWurst](/ChristophWurst) added the [pending documentation](/nextcloud/server/labels/pending%20documentation) label [11 seconds ago](#event-3339654979)

Please add this for all of these deprecations and removals :pray: 

And we should maybe also add this to https://github.com/nextcloud/server/issues/20953

File: core/src/OC/util.js [hunk 1]
```diff
@@ -114,6 +114,7 @@ export default {
 	 * @returns {string} timestamp formatted as requested
 	 */
 	formatDate: function(timestamp, format) {
+		console.warn('OC.Util.formatDate is deprecated and will be removed in Nextcloud 21')
```
Comment (juliushaertl) [inline core/src/OC/util.js:117]: Maybe add a hint to use moment directly

**Review by ChristophWurst — APPROVED** [review]

**Review by juliusknorr — APPROVED** [review]
Otherwise :+1:

**Comment by ChristophWurst** [comment]
Documented with the other changes from #20953

**Pull request merged**